webview.go

```diff
@@ -245,62 +254,43 @@ func (w *webview) Bind(name string, f interface{}) error {
 	if v.Kind() != reflect.Func {
 		return errors.New("only functions can be bound")
 	}
-	// f must return either value and error or just error
-	if n := v.Type().NumOut(); n > 2 {
-		return errors.New("function may only return a value or a value+error")
+
+	if v.Type().NumIn() > 0 && v.Type().In(0).Kind() != reflect.String {
+		return errors.New("first argument must be a string for the request id")
```

Comment:
At least this will give a warning about `id string` needing to be first argument.